packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -87,93 +47,40 @@ pipelines:
-                - pnpm exec nx-cloud record -- pnpm exec nx format:check
-                - pnpm exec nx affected --target=lint & pnpm exec nx affected --target=test & pnpm exec nx affected --target=build
-                - pnpm exec nx-cloud stop-all-agents
-          - step: *agent
-          - step: *agent
-          - step: *agent
+                - npm ci
+                # uncomment to enable task distribution
+                # - npx nx-cloud start-ci-run --distributes-on="5 linux-medium"
+                - npx nx-cloud record -- nx format:check
+                - npx nx affected --target=lint,test,build --parallel=3
 "
 `;
 
 exports[`CI Workflow generator with npm should generate circleci CI config 1`] = `
 "version: 2.1
 
 orbs:
-  nx: nrwl/nx@1.6.1
+  nx: nrwl/nx@1.6.2
 
 jobs:
-  agent:
-    docker:
-      - image: cimg/node:lts-browsers
-    parameters:
-      ordinal:
-        type: integer
-    steps:
-      - checkout
-
-      - run:
-          name: Install PNPM
-          command: npm install --prefix=$HOME/.local -g pnpm@8.2.0
-      - run:
-          name: Install dependencies
-          command: pnpm install --frozen-lockfile
-      - run:
-          name: Start the agent << parameters.ordinal >>
-          command: pnpm exec nx-cloud start-agent
-          no_output_timeout: 60m
   main:
     docker:
       - image: cimg/node:lts-browsers
-    environment:
-      NX_CLOUD_DISTRIBUTED_EXECUTION: 'true'
     steps:
       - checkout
-
-      - run:
-          name: Install PNPM
-          command: npm install --prefix=$HOME/.local -g pnpm@8.2.0
-      - run:
-          name: Install dependencies
-          command: pnpm install --frozen-lockfile
+      - run: npm ci
       - nx/set-shas:
           main-branch-name: 'main'
-      - run:
-          name: Initialize the Nx Cloud distributed CI run
-          command: pnpm exec nx-cloud start-ci-run --stop-agents-after="build" --agent-count=3
-      - run:
-          name: Check format
-          command: pnpm exec nx-cloud record -- pnpm exec nx format:check --base=$NX_BASE --head=$NX_HEAD
-      - run:
-          name: Run lint
-          command: pnpm exec nx affected --base=$NX_BASE --head=$NX_HEAD --target=lint --parallel=3
-      - run:
-          name: Run test
-          command: pnpm exec nx affected --base=$NX_BASE --head=$NX_HEAD --target=test --parallel=3 --ci --code-coverage
-      - run:
-          name: Run build
-          command: pnpm exec nx affected --base=$NX_BASE --head=$NX_HEAD --target=build --parallel=3
-      - run:
-          name: Stop all agents
-          command: pnpm exec nx-cloud stop-all-agents
-          when: always
+      # uncomment to enable task distribution
+      # - run: npx nx-cloud start-ci-run --distributes-on="5 linux-medium"
+      - run: npx nx-cloud record -- nx format:check --base=$NX_BASE --head=$NX_HEAD
+      - run: npx nx affected --base=$NX_BASE --head=$NX_HEAD --target=lint,test,build --parallel=3
```

Comment:
-t

Reply:
Is the passing of base and head really necessary here? It should be read from the env automatically? Is there something specific about circle that doesn’t allow it to work?

Reply:
This only works in GitHub. Other CI providers allow passing variables between steps but do not add them to process environments. We had tested this in the past.

Reply:
Ah ok thanks